samples/tiny_http/src/main.rs

```diff
@@ -0,0 +1,58 @@
+extern crate tiny_http;
+extern crate multipart;
+
+use multipart::server::{Multipart, Entries, SaveResult};
+use tiny_http::{Response, StatusCode, Request};
+fn main() {
+    // Starting a server on `localhost:80`
+    let server = tiny_http::Server::http("localhost:80").unwrap();
```

Comment:
This `.unwrap()` is fine but `.expect("Could not bind localhost:80")` would be better. :smile: